astropy/timeseries/downsample.py

```diff
@@ -93,7 +116,7 @@ def aggregate_downsample(
         parameter will be ignored.
     aggregate_func : callable, optional
         The function to use for combining points in the same bin. Defaults
-        to np.nanmean.
+        to an internal implementation of NumPy `nanmean` function.
```

Comment:
backticks are interpreted by Sphinx as a reference which doesn't resolve. This needs to be addressed to fix doc builds. I also suggest not refering to NumPy in text either because there's no obvious relation between np.nanmean and your implementation
```suggestion
        to an internal implementation of nanmean.
```